### PR TITLE
Refactor Create Event Page #442

### PR DESCRIPTION
**Pull request by Boorclark**
Fixes Issue #442 

Changes Made:
 - Added conditional so users who are not CELTS Admins or Program managers cannot see the edit event tab or track volunteers
 - Added a View Event Tab
 - Got  the RSVP flash messages to display properly
 - Changed the way the RSVP buttons are accessed in the JS (no longer in a form)
 - Deleted createAllVolunteerEvent.html
 - Renamed createSingleEvent to createEvent across application
 - Cleaned up unneeded conditionals in the createEvent.html file
 - Added RSVP button in viewEvent.html

--------------------------------------------------------------------------------------------------------------------------------------------

Files Changed:
- app/controllers/admin/routes.py
- app/controllers/main/routes.py
- app/static/js/event_list.js
- app/templates/admin/createAllVolunteerEvent.html
- app/templates/admin/createEvent.html
- app/templates/eventView.html
- app/templates/events/trackVolunteers.html
- database/base_data.py

### CODE REVIEW

**Comment by BrianRamsay** [comment]
A number of conflicts to resolve.

File: app/templates/eventView.html [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/js/bootstrap-select.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+    <script src="/static/js/event_list.js"></script>
+{% endblock %}
+
+{% block styles %}
+    {{super()}}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/css/bootstrap-select.css" />
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+{% endblock %}
```
Comment (BrianRamsay) [inline app/templates/eventView.html]: Do you need bootstrap-select and jquery-ui for this page?

File: app/controllers/admin/routes.py [hunk 2]
```diff
@@ -144,10 +144,12 @@ def eventDisplay(eventId):
     userHasRSVPed = EventRsvp.get_or_none(EventRsvp.user == g.current_user, EventRsvp.event == event)
     isPastEvent = (datetime.now() >= datetime.combine(event.startDate, event.timeStart))
     program = event.singleProgram
+    isProgramManager = hasPrivilege(g.current_user,program)
```
Comment (BrianRamsay) [inline app/controllers/admin/routes.py]: Merge in development. This will need to change.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/controllers/admin/routes.py (hunk 2 above)
Comment (Boorclark) [reply app/controllers/admin/routes.py]: resolved

File: app/templates/eventView.html (hunk 1 above)
Comment (Boorclark) [reply app/templates/eventView.html]: resolved

**Comment by BrianRamsay** [comment]
Merge conflict from a recent development change

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**